### ISSUE TITLE
remove color tags on log entries

### ISSUE THREAD

**Issue by jcohenadad** [neuropoly/spinalcordtoolbox#1035]
things like: [0m, [94m, etc.

**Comment by jcohenadad** [comment]
color markers still present in some log files:

~~~
test ran at 161231120310
sct_label_vertebrae_vanderbilt_05_161231120221_64278/output.log
====================================================================================================
sct_label_vertebrae  -c t2 -laplacian 0 -i /Users/julien/data/sct_test_function/vanderbilt_05/t2/t2.nii.gz -t /Users/julien/code/sct/data/PAM50/ -v 2 -s /Users/julien/data/sct_test_function/vanderbilt_05/t2/t2_seg_manual.nii.gz -r 1 -denoise 0 -ofolder sct_label_vertebrae_vanderbilt_05_161231120221_64278/
====================================================================================================


/Users/julien/code/sct/bin/sct_launcher: line 43: CE.app/Contents/helpers/pydev: No such file or directory
--
Spinal Cord Toolbox (jca_issue_1083/c6a5d01fc23d6e5607ae184a6f6b150cb2073d90)
Running /Users/julien/code/sct/scripts/sct_label_vertebrae.py -c t2 -laplacian 0 -i /Users/julien/data/sct_test_function/vanderbilt_05/t2/t2.nii.gz -t /Users/julien/code/sct/data/PAM50/ -v 2 -s /Users/julien/data/sct_test_function/vanderbilt_05/t2/t2_seg_manual.nii.gz -r 1 -denoise 0 -ofolder sct_label_vertebrae_vanderbilt_05_161231120221_64278/
Check folder existence...
Check folder existence...

Create temporary folder...

Create temporary folder...

Copying input data to tmp folder...
[94msct_convert -i /Users/julien/data/sct_test_function/vanderbilt_05/t2/t2.nii.gz -o tmp.161231120222_343257/data.nii[0m
[94msct_convert -i /Users/julien/data/sct_test_function/vanderbilt_05/t2/t2_seg_manual.nii.gz -o tmp.161231120222_343257/segmentation.nii.gz[0m

Create label to identify disc...

Straighten spinal cord...
[94msct_straighten_spinalcord -i data.nii -s segmentation.nii.gz -r 0 -qc 0[0m

Resample to 0.5mm isotropic...
main
[94msct_resample -i data_straight.nii -mm 0.5x0.5x0.5 -x linear -o data_straightr.nii[0m
Running /Users/julien/code/sct/scripts/sct_resample.py -i data_straight.nii -mm 0.5x0.5x0.5 -x linear -o data_straightr.nii

Load data...
pixdim: (1.0000002, 0.9765625, 0.9765625)
shape: (59, 71, 107)

Calculate new dimensions...
new shape: (118, 139, 209)
affine matrix:
[[ -1.00000024  -0.           0.          31.9433651 ]
[ -0.           0.9765625   -0.         -28.88208771]
[  0.           0.           0.9765625  -87.23549652]
[  0.           0.           0.           1.        ]]

Calculate affine transformation...
transfo:
[[ 0.5         0.          0.         -0.25      ]
[ 0.          0.51079137  0.         -0.24460432]
[ 0.          0.          0.51196172 -0.24401914]
[ 0.          0.          0.          1.        ]]
/Users/julien/code/sct/python/lib/python2.7/site-packages/nipy/io/files.py:145: FutureWarning: Default `strict` currently False; this will change to True in a future version of nipy
ni_img = nipy2nifti(img, data_dtype = io_dtype)

Done! To view results, type:
fslview data_straightr.nii &

Apply straightening to segmentation...
main
[94msct_apply_transfo -i segmentation.nii.gz -d data_straightr.nii -w warp_curve2straight.nii.gz -o segmentation_straight.nii.gz -x linear[0m
Running /Users/julien/code/sct/scripts/sct_apply_transfo.py -i segmentation.nii.gz -d data_straightr.nii -w warp_curve2straight.nii.gz -o segmentation_straight.nii.gz -x linear

Parse list of warping fields...
Transfo #0: warp_curve2straight.nii.gz

Get dimensions of data...
256 x 256 x 60 x 1

Apply transformation...
[94misct_antsApplyTransforms -d 3 -i segmentation.nii.gz -o segmentation_straight.nii.gz -t warp_curve2straight.nii.gz -r data_straightr.nii -n Linear[0m

Done! To view results, type:
fslview data_straightr.nii segmentation_straight.nii.gz &

main
[94msct_maths -i segmentation_straight.nii.gz -thr 0.5 -o segmentation_straight.nii.gz[0m
Running /Users/julien/code/sct/scripts/sct_maths.py -i segmentation_straight.nii.gz -thr 0.5 -o segmentation_straight.nii.gz
WARNING: File segmentation_straight.nii.gz already exists. Deleting it.

Done! To view results, type:
fslview segmentation_straight.nii.gz &


Look for template...
Path template: /Users/julien/code/sct/data/PAM50/

Open template and vertebral levels...

Disc values from template: [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19]
Z-values for each disc: [962, 939, 908, 871, 834, 801, 770, 736, 693, 647, 601, 552, 501, 450, 397, 343, 290, 232, 169, 106, 56]
Distances between discs (in voxel): [23, 31, 37, 37, 33, 31, 34, 43, 46, 46, 49, 51, 51, 53, 54, 53, 58, 63, 63, 50]

Detect C2/C3 disk...
.. Peak found: z=148 (correlation = 0.301828628304)

Detect intervertebral discs...
Current disc: 2 (z=133). Direction: superior
.. Peak found: z=1 (correlation = 0.404910014937)
Current disc: 1 (z=165). Direction: superior
.. correcting factor: 1.0
.. Switching to inferior direction.
Current disc: 3 (z=96). Direction: inferior
.. Peak found: z=-2 (correlation = 0.327588755728)
.. correcting factor: 1.04054054054
Current disc: 4 (z=55). Direction: inferior
.. Peak found: z=-10 (correlation = 0.378433245924)
.. correcting factor: 1.13513513514
Current disc: 5 (z=8). Direction: inferior
.. Peak found: z=5 (correlation = 0.345407673294)
/Users/julien/code/sct/python/lib/python2.7/site-packages/matplotlib/axes/_base.py:3045: UserWarning: Attempting to set identical bottom==top results
in singular transformations; automatically expanding.
bottom=-0.5, top=-0.5
'bottom=%s, top=%s') % (bottom, top))
.. correcting factor: 1.09377559378
Adding top disc based on adjusted template distance: #0
.. approximate distance: 25

Un-straighten labeling...
main
[94msct_apply_transfo -i segmentation_straight_labeled.nii.gz -d segmentation.nii.gz -w warp_straight2curve.nii.gz -o segmentation_labeled.nii.gz -x nn[0m
Running /Users/julien/code/sct/scripts/sct_apply_transfo.py -i segmentation_straight_labeled.nii.gz -d segmentation.nii.gz -w warp_straight2curve.nii.gz -o segmentation_labeled.nii.gz -x nn

Parse list of warping fields...
Transfo #0: warp_straight2curve.nii.gz

Get dimensions of data...
118 x 139 x 209 x 1

Apply transformation...
[94misct_antsApplyTransforms -d 3 -i segmentation_straight_labeled.nii.gz -o segmentation_labeled.nii.gz -t warp_straight2curve.nii.gz -r segmentation.nii.gz -n NearestNeighbor[0m

Done! To view results, type:
fslview segmentation.nii.gz segmentation_labeled.nii.gz &


Clean labeled segmentation (correct interpolation errors)...
[94msct_maths -i segmentation_labeled.nii.gz -mul segmentation.nii.gz -o segmentation_labeled_mul.nii.gz[0m
[94msct_maths -i segmentation_labeled.nii.gz -dilate 2 -o segmentation_labeled_dilate.nii.gz[0m
[94msct_maths -i segmentation_labeled_mul.nii.gz -bin 0 -o segmentation_labeled_mul_bin.nii.gz[0m
WARNING: File segmentation_labeled.nii.gz already exists. Deleting it.

Label discs...

Generate output files...
File created: sct_label_vertebrae_vanderbilt_05_161231120221_64278/t2_seg_manual_labeled.nii.gz
File created: sct_label_vertebrae_vanderbilt_05_161231120221_64278/t2_seg_manual_labeled_discs.nii.gz
File created: sct_label_vertebrae_vanderbilt_05_161231120221_64278/warp_curve2straight.nii.gz
File created: sct_label_vertebrae_vanderbilt_05_161231120221_64278/warp_straight2curve.nii.gz
File created: sct_label_vertebrae_vanderbilt_05_161231120221_64278/straight_ref.nii.gz

Remove temporary files...
[94mrm -rf tmp.161231120222_343257/[0m

Done! To view results, type:
fslview /Users/julien/data/sct_test_function/vanderbilt_05/t2/t2.nii.gz sct_label_vertebrae_vanderbilt_05_161231120221_64278/t2_seg_manual_labeled -l Random-Rainbow -t 0.5 &

WARNING: RMSE = 7.87411575634 > 2
WARNING: Max distance = 21.4220132302 > 4
~~~

**Comment by jcohenadad** [comment]
this is caused by the use of print instead of printv inside run():
sct_utils@l78:
~~~
def run(cmd, verbose=1, error_exit='error', raise_exception=False):
    if verbose==2:
        printv(sys._getframe().f_back.f_code.co_name, 1, 'process')
    if verbose:
        print(bcolors.blue+cmd+bcolors.normal)
~~~

**Comment by jcohenadad** [comment]
fixed by PR #1085
